core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkspaceManagerResourceMonitorRecordComponent.scala

```diff
@@ -13,19 +14,26 @@ object WorkspaceManagerResourceMonitorRecord {
     type JobType = Value
     val AzureLandingZoneResult: Value = Value("AzureLandingZoneResult")
   }
+
+  implicit sealed class JobStatus(val complete: Boolean)
+
+  case object Complete extends JobStatus(true)
+
+  case object Incomplete extends JobStatus(false)
```

Comment:
This looks a bit fishy to me. Your sum type already encodes whether or not the job's complete.
Would it make sense to encode "finished" as an `Option[Instant]` and an failure message as`Option[String]` on the job record itself?